index.html

```diff
@@ -1,28 +1,101 @@
+                <li><strong>1962</strong> - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</li>
+                <li><strong>1983</strong> - helps seven African countries dramatically increase their maize and sorghum yields</li>
+                <li><strong>1984</strong> - becomes a distinguished professor at Texas A&M University</li>
+                <li><strong>2005</strong> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds."</li>
+                <li><strong>2009</strong> - dies at the age of 95.</li>
+            </ul>
+            <blockquote>
+                <p>
+                <em>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."
 
-<body>
-  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-  Hello World, we are software developers!
-  <!--
+                </p>
+                <cite><em>-- Indian Prime Minister Manmohan Singh</em></cite>
+            </blockquote>
+            <h3><strong>If you have time, you should read more about this incredible human being on his <a href="https://en.wikipedia.org/wiki/Norman_Borlaug" id="tribute-link" target="_blank"> Wikipedia entry.</a></strong></h3>
 
-    Hello Camper!
+        </div>
+        
+        
+    </div>
+    </body>
+</html><html>
+    <head>
+        <link href="style.css" rel="stylesheet" >
+        <title>Tribute Page</title>
+    </head>
+    
+    <body>
+        <div id="main">
+        <header>
+            <h1 id="title">Dr Norman Borlaug</h1>
+            <p id="description" >The man who saved a billion lives</p>
+        </header>
+        <figure id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" />
+            <figcaption id="img-caption">Dr. Norman Borlaug, third from left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger</figcaption>
+        </figure>
 
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
+        <div class="history" id="tribute-info">
```

Comment:
Instead of a div, you could a section here.

Reply:
- [ ] Good job phrasing your comment. but you have to also provide a reason why you are requesting this particular change and also it's better to be more polite you can say _Kindly use this..._

Reply:
Feedback appreciated

Reply:
Add your updated comments here.

Reply:
Kindly use the section tag instead of div. This is semantically more accurate than using a div.